doc/manual/en/ch11_configuration.tex

```diff
@@ -663,6 +663,10 @@ \section{Look / Screen Layout}\label{sec:interface-appearance}
   {\bf Box}: Draws boxes around each InfoBox. \\
   {\bf Tab}: Draws a tab at the top of the InfoBox across the 
   title. \\
+\item[Cursor size*]  Control the size of the cursor. Default is 2.
```

Comment:
Question: do we document features that depend on the build target?

Reply:
Yes, please - maybe with a footnote?